### ISSUE TITLE
Can you provide a training code?

### ISSUE THREAD

**Issue by LucasShi1101** [clement-bonnet/text-to-pose#5]
I am reproducing your project now, but I found that there is no training code. Can you please provide a training method?

**Comment by clement-bonnet** [comment]
Hi, I do not have access to the training code anymore but you can assume a pretty simple code. The T2P model is instantiated and then trained with Adam on that [loss function](https://github.com/clement-bonnet/text-to-pose/blob/main/t2p/model.py#L135).
Please refer to this issue for discussion around training: #4
